static/app/utils/analytics/profilingAnalyticsEvents.tsx

```diff
@@ -13,6 +14,7 @@ export type ProfilingEventParameters = {
 
 export const profilingEventMap: Record<EventKey, string> = {
   'profiling_views.landing': 'Profiling Views: Landing',
+  'profiling_views.onboarding': 'Profiling Views: Onboarding',
```

Comment:
I think you have an older branch. Can you pull master and update this as needed?

Reply:
I think git is doing something fucky here. The base branch is up to date and was rebased onto master. This file hasn't changed and doesn't have changes on the base branch